docs: additional capabilities pages DOC-1144 (#3550)
spectrocloud/librarium: docs/docs-content/getting-started/additional-capabilities/vmo.md

lifecycle management capabilities as Palette applies to containers, including backups.

VMO uses the CNCF project [KubeVirt](https://kubevirt.io) to manage VMs as Kubernetes pods, ensuring complete mapping
between the VM and Kubernetes concepts. This solution also has near complete feature parity with
[VMware vSphere](https://www.vmware.com/products/vsphere.html), including capabilities such as live migration.

Palette VMO can be used on edge hosts, giving the ability to deploy VM workloads at the edge without the overhead of a
hypervisor layer. This is achieved by leveraging [Canonical MAAS](https://maas.io). Additionally, VMO can also be used
in self-hosted, airgapped, and in our SaaS environments. Learn more on the
[Virtual Machine Orchestrator](../../vm-management/vm-management.md) page.

## Resources

To learn more about Palette VMO, review the [Architecture](../../vm-management/architecture.md) page to learn about the
components involved in enabling VMO for your infrastructure. Then, review the
[Create a VMO Profile](../../vm-management/create-vmo-profile.md) guide to prepare everything you need to deploy your

> first VMO cluster.

Reviewer: ```suggestion
first cluster with VMO.
```

> Check out the following video for a tour of Palette's Virtual Machine Orchestrator (VMO) capability, showing how you can

Reviewer: ```suggestion
Check out the following video for a tour of Palette's Virtual Machine Orchestrator (VMO) capability. It shows how you can
```

> model, deploy and manage VM workloads alongside containers in all your clusters and environments.

Reviewer: ```suggestion
model, deploy, and manage VM workloads alongside containers in your clusters.
```
maybe?
